examples/open_loop_control_urdfbot/generate_actions.py

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+# generate series of actions for fetch agent on different parts such as moving, arm and gripper.
+
+import argparse
+import os
+import numpy as np
+import pandas as pd
+
+# fetch arm poses come from https://github.com/StanfordVL/iGibson/blob/master/igibson/robots/fetch.py#L100
+arm_poses = {
+    "init": np.array([0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]),
+    "default": np.array([-1.17079, 1.47079, 0.4, 1.67079, 0.0, 1.57079, 0.0]),
+    "vertical": np.array([-0.94121, -0.64134, 1.55186, 1.65672, -0.93218, 1.53416, 2.14474]),
+    "diagonal15": np.array([-0.95587, -0.34778, 1.46388, 1.47821, -0.93813, 1.4587, 1.9939]),
+    "diagonal30": np.array([-1.06595, -0.22184, 1.53448, 1.46076, -0.84995, 1.36904, 1.90996]),
+    "diagonal45": np.array([-1.11479, -0.0685, 1.5696, 1.37304, -0.74273, 1.3983, 1.79618]),
+    "horizontal": np.array([-1.43016, 0.20965, 1.86816, 1.77576, -0.27289, 1.31715, 2.01226]),
+}
+
+
+def add_actions(move_forward=0.0, move_right=0.0, gripper_state=50.0, arm_current="init", arm_next="init", arm_alpha=0.0):
```

Comment:
Pass in a dictionary of blend weights rather than arm_current, arm_next, and a single blending value. Return an action (not a `DataFrame`, convert it to a DataFrame after it is returned) because then it's a cleaner simpler general-purpose function.

```
def get_action(..., arm_pose_blend_weights={}):
    action = {}

    action["joint.wheel_joint_r"] = move_forward - move_right
    action["joint.wheel_joint_l"] = move_forward + move_right

    # ...

    return action
```

Reply:
Im not sure I if I get it. You mean `arm_pose_blend_weights` would be the same as current `arm_pose` or `arm_pose` in dict format? Or like following? 
```
arm_pose_blend_weights = { arm_current="init", arm_next="init", arm_alpha=0.0 }
```
or